### PR TITLE
updated to tf12

### PR DESCRIPTION
**Pull request by rajhansh1994**
Updated terraform version from 11 to 12

### CODE REVIEW

**Comment by rajhansh-seyfert** [comment]
Forgot to add reviewers

**Comment by rajhansh1994** [comment]
Please review the same

File: main.tf [hunk 1]
```diff
@@ -14,25 +15,26 @@ data "aws_iam_policy_document" "repo" {
       "ecr:GetAuthorizationToken",
       "ecr:GetDownloadUrlForLayer",
       "ecr:GetRepositoryPolicy",
-      "ecr:ListImages"
+      "ecr:ListImages",
     ]
     principals {
       identifiers = [
```
Comment (vchinnakotla) [inline main.tf]: use flatten function to make it as a single list. or TF12 errors out.

**Review by vchinnakotla — CHANGES_REQUESTED** [review]
see the inline comments

File: main.tf (hunk 1 above)
Comment (vchinnakotla) [reply main.tf]: I asked for the identifiers, as the list contains a list inside called var.cross_account_users.
Please remove for the actions and add it to identifiers.

**Pull request merged**